Gopkg.lock

```diff
@@ -360,6 +360,14 @@
   revision = "4b7aa43c6742a2c18fdef89dd197aaae7dac7ccd"
   version = "1.0.1"
 
+[[projects]]
+  digest = "1:8f41bbca0d6435bb54d7c090336f7693942cda30f9126eaf3959b8df48721ac3"
+  name = "github.com/myesui/uuid"
```

Comment:
There are already 2 vendored uuid packages, `github.com/google/uuid` and `github.com/pborman/uuid`, can we use one of those?

Reply:
Will use google